test/e2e_node/README-windows.md

```diff
@@ -0,0 +1,37 @@
+# Windoes Node End-To-End (e2e) tests
+
+Node e2e tests are component tests meant for testing the Kubelet code on a custom host environment, they are now supported on the Windows platform and can be executed locally, with remote execution planned as the next step.
```

Comment:
please document this feature tag: https://github.com/kubernetes/kubernetes/blob/39f1c90ac014828803a156b40d49f0c58fb7384a/test/e2e/feature/feature.go#L568

Reply:
(do both - replace TODO in https://github.com/kubernetes/kubernetes/blob/39f1c90ac014828803a156b40d49f0c58fb7384a/test/e2e/feature/feature.go#L567 with the actual description. And add note about that tag into this document